packages/react-native-usb/src/ReactNativeUsb.types.ts

```diff
@@ -81,5 +81,5 @@ export interface WebUSBDevice {
 }
 
 export interface OnConnectEvent extends Event {
-    device: WebUSBDevice;
+    device: WebUSBDevice | null | undefined;
```

Comment:
Why undefined? I'd prefer to use `null` only if it's not required for some third party type compatibility?

Reply:
Just code style of course. I understand this changes nothing with how the app runs.. Feel free to resolve if you disagree.

Reply:
I was not sure what could be here, null/undefined come from Expo/RN native part so I better put both variants here, so it's explicitely defined at least.